apps/photos/src/components/Chip.tsx

```diff
@@ -1,10 +1,6 @@
-import { Box, styled } from '@mui/material';
+import { Button, styled } from '@mui/material';
 import { CSSProperties } from '@mui/material/styles/createTypography';
 
-export const Chip = styled(Box)(({ theme }) => ({
+export const Chip = styled(Button)(({ theme }) => ({
```

Comment:
@jubitjohn, can you confirm the chip colour and padding are correct as per  
figma https://www.figma.com/file/SYtMyLBs5SAOkTbfMMzhqt/ente-Visual-Design?type=design&node-id=8119-59512&t=nQxLsSFPN6BGdwSW-4

Reply:
Okay, I have checked and made the corrections